### ISSUE TITLE
Create, update, and maintain internal alumni DB

### ISSUE THREAD

**Issue by mysticuno** [Dsan10s/NDWebsite#11]
There exists a Google doc that has some information about alumni, but some of that info is either out-of-date, limited, or non-existent.

It would be cool if we could have an internal DB where brothers (using certificates) can easily get information about our alumni. It'd be cool if we could integrate this with our family trees as well. We could work with our current and past Alumni Secretaries to try and gain as much information as we can. We can also talk with brothers who are Tech Callers since they are likely to have information about alumni of the house.

There was an email that was sent out last summer that reached out to fraternity men asking them to call alumni in the hopes that they would have better success at securing donations to MIT from alumni since the callers had a connection with them. I can't remember who organized the calling, but I can look through my emails later (or if someone else can find it that'd be great) and try contacting him for more info.

**Comment by mysticuno** [comment]
I was talking to EC earlier today and Brian mentioned about having something like an alumni connection tab, someplace where we can browse and highlight prominent alumni. 

There could be a main page that can show previous classes, and basically allow for people to browse alumni and see information about them such as lineage and and other info that we currently show for brothers. It'd would be really cool if we could also have this page feature our alumni of the week. I think this really has potential, and since it's mainly providing information I don't think it would be terribly hard to implement.

**Comment by Dsan10s** [comment]
This is something I've wanted (specifically the Alumni of the week thing) for quite a long time, but from past experience, it is extremely difficult to get Alumni Sec to send you the Alumni of the week info.  If Alumni sec just put their Alumni of the week on a google doc every week, that would be even better.

**Comment by mysticuno** [comment]
Yeah, I've noticed that it's been historically difficult to get Alumni Sec to do much of anything, but I really believe that it's something that has a lot of potential and that deserves putting a lot of work into it, even if it means constantly pestering Alumni Sec for info if that's the only way to get it from him.

I also agree that  a Google Doc would be much better than just one document that lives on the Alumni Sec's computer. It could also be a living document so we could see previous alumni of the week. Furthermore, if AS was really proactive, he could set up a queue for alumni of the week (like maybe have them planned of for the next two weeks or so), and then it'd be really easy for us to update the alumni of the week.
